keccak-air/Cargo.toml

```diff
@@ -39,3 +39,4 @@ tracing-forest = { version = "0.1.6", features = ["ansi", "smallvec"] }
 # We should be able to enable p3-maybe-rayon/parallel directly; this just doesn't
 # seem to work when using cargo with the -p or --package option.
 parallel = ["p3-maybe-rayon/parallel"]
+asm = ["p3-sha256/asm"]
```

Comment:
This should probably be called `sha2_asm` (or similar) to make it clear that it only relates to that one very specific use of assembly in our codebase.